fhir-database-utils/src/main/java/com/ibm/fhir/database/utils/query/node/ExpNodeParser.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.database.utils.query.node;
+
+import java.util.Stack;
+
+/**
+ * Stack based parser for processing expression nodes
+ */
+public class ExpNodeParser {
+
+    private final Stack<ExpNode> stack = new Stack<>();
+
+    /**
+     * Push a node representing the column reference onto the stack
+     * @param columnName
+     */
+    public void column(String columnName) {
+        push(new ColumnExpNode(columnName));
+    }
+
+    public void literal(String str) {
+
```

Comment:
maybe a // NOP comment for these methods if that's the intent

Reply:
removed